docs/content/docs/user-docs/leader-election.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Leader election"
+description: "Configure leader election for ACK controllers"
+lead: "How to enable leader election for ACK controllers"
+draft: false
+menu:
+  docs:
+    parent: "getting-started"
+weight: 60
+toc: true
+---
+
+In a Kubernetes cluster, multiple instances of a controller maybe running
+simultaneously to manage resources and perform tasks. However, to avoid conflicts
+and ensure proper resource management, it's necessary to designate one instance
+as the leader, which takes responsibility for executing certain operations while
+the other instances remain passive. In the event that the leader instance fails,
+[leader election][leader-election] ensures the seamless transition of leadership
+to another healthy instance.
+
+In ACK land, by default, the feature of leader election is disabled. To utilize
+it, users can configure leader election using either Helm chart values or
+controller flags. Once enabled, you have the flexibility to scale the default
+deployment of ACK controllers from a single replica (1) to a higher number.
```

Comment:
`or controller flags` is implied when using the Helm chart. We don't advise installing directly from manifests, so our docs don't reference those.